cypress/integration/event-signup-flow.spec.js

```diff
@@ -211,6 +211,7 @@ describe("Feature - Event sign up : Tests execution date and time : " + new Date
 						});
 						eventSignup.getNextStep().click();
 						eventSignup.getBackButton().should("exist").should("have.text", "Back");
+						cy.verifyMobileFieldandDisplayedValue(this.testData.phoneNumber);
```

Comment:
Typo in the method name there, should be upper case `And`

Reply:
aah, changed it, thanks